boot/boot_serial/src/boot_serial.c

```diff
@@ -255,6 +262,10 @@ bs_upload(char *buf, int len)
     static off_t off_last = -1;
     struct flash_sector sector;
 #endif
+#ifdef MCUBOOT_DOWNGRADE_PREVENTION
+    struct image_header* inc_hdr = NULL;
+    struct image_header st_hdr;
+#endif /*MCUBOOT_DOWNGRADE_PREVENTION*/
```

Comment:
Also, sorry to nit, but please put spaces between the comment characters and the symbol. I notice there are a few others that have snuck in without the spaces, but the original ones do have the spaces.